lib/block-supports/typography.php

```diff
@@ -621,3 +621,41 @@ function gutenberg_get_typography_font_size_value( $preset, $settings = array()
 	remove_filter( 'render_block', 'wp_render_typography_support' );
 }
 add_filter( 'render_block', 'gutenberg_render_typography_support', 10, 2 );
+
+/**
+ * Filters the block type arguments to stabilize experimental block supports.
```

Comment:
Do you reckon this migration will be tied to 6.7, or a general deprecation? 

I guess what I'm really asking is, should the filter be in a compat folder?

Reply:
Oh, yes, a `compat` folder is a better idea since the idea is to remove this once 6.7+ is the required WP version for Gutenberg 👍

I'll move that around tomorrow. Thanks!

Reply:
Done in https://github.com/WordPress/gutenberg/pull/63401/commits/71a0a964bc41625a5e9c67cf3cdec54445253f58 👍